UPGRADE-7.0.md

```diff
@@ -4,12 +4,55 @@ UPGRADE FROM 6.4 to 7.0
 Symfony 6.4 and Symfony 7.0 will be released simultaneously at the end of November 2023. According to the Symfony
 release process, both versions will have the same features, but Symfony 7.0 won't include any deprecated features.
 To upgrade, make sure to resolve all deprecation notices.
-Read more about this in the [Symfony documentation](https://symfony.com/doc/current/setup/upgrade_major.html).
+Read more about this in the [Symfony documentation](https://symfony.com/doc/7.0/setup/upgrade_major.html).
+
+Symfony 7.0 introduced many native return and property types. Read [the announcement blogpost](https://symfony.com/blog/symfony-7-0-type-declarations)
+on how to quickly make your code compatible.
+
+Table of Contents
+-----------------
+
+Bundles
+ * [FrameworkBundle](#FrameworkBundle)
+ * [SecurityBundle](#SecurityBundle)
+ * [TwigBundle](#TwigBundle)
+
+Bridges
+ * [DoctrineBridge](#DoctrineBridge)
+ * [MonologBridge](#MonologBridge)
+ * [ProxyManagerBridge](#ProxyManagerBridge)
+
+Components
+ * [Cache](#Cache)
+ * [Config](#Config)
+ * [Console](#Console)
+ * [DependencyInjection](#DependencyInjection)
+ * [DomCrawler](#DomCrawler)
+ * [ExpressionLanguage](#ExpressionLanguage)
+ * [Filesystem](#Filesystem)
+ * [Form](#Form)
+ * [HttpFoundation](#HttpFoundation)
+ * [HttpClient](#HttpClient)
+ * [HttpKernel](#HttpKernel)
+ * [Lock](#Lock)
+ * [Mailer](#Mailer)
+ * [Messenger](#Messenger)
+ * [Mime](#Mime)
+ * [PropertyAccess](#PropertyAccess)
+ * [Routing](#Routing)
+ * [Security](#Security)
+ * [Serializer](#Serializer)
+ * [Templating](#Templating)
+ * [Translation](#Translation)
+ * [Validator](#Validator)
+ * [VarDumper](#VarDumper)
+ * [Workflow](#Workflow)
+ * [Yaml](#Yaml)
 
 Cache
 -----
 
- * Add parameter `$isSameDatabase` to `DoctrineDbalAdapter::configureSchema()`
+ * Add argument `\Closure $isSameDatabase` to `DoctrineDbalAdapter::configureSchema()`
```

Comment:
Parameter is the correct term.

A method signature contains _parameters_, you call that method with _arguments_.